pkgs/development/compilers/ghc/8.4.2.nix

```diff
@@ -24,7 +24,12 @@
   # platform). Static libs are always built.
   enableShared ? !targetPlatform.useAndroidPrebuilt
 
+, # Whetherto build terminfo.
+  enableTerminfo ? !targetPlatform.isWindows
+
 , version ? "8.4.2"
+, ghcFlavour ? ""
```

Comment:
Can you please add a small comment here explaining that `""` means "use ghc's default choice"?

Reply:
will do.